src/app/games/coerceo/coerceo.component.ts

```diff
@@ -44,8 +47,11 @@ export class CoerceoComponent extends TriangularGameComponent<CoerceoRules,
         this.rules = CoerceoRules.get();
         this.node = this.rules.getInitialNode();
         this.availableAIs = [
-            new CoerceoMinimax(),
-            new CoerceoPiecesThreatTilesMinimax(),
+            new Minimax('Pieces > Threats > Tiles',
+                        this.rules,
```

Comment:
ah ! this.rules et CoerceoRules.get() ! CHOISIR !

Reply:
yep, ici le this.rules est présent donc autant l'utiliser, je prends donc celui là !